ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/mcs-client.j2

```diff
@@ -0,0 +1,24 @@
+{# MCS Client #}
+{% if mcs_client is arista.avd.defined %}
+
+## MCS client Summary
+{%     if shut is arista.avd.defined(true) %}
+MCS client is shutdown
```

Comment:
there is no variable `shut`. Should probably be `mcs_client.shutdown.
Also missing a blank line between the if and the text.
```suggestion
{%     if mcs_client.shutdown is arista.avd.defined(true) %}

MCS client is shutdown
```